doc/develop/modules.rst

```diff
@@ -569,6 +569,36 @@ Build files located in a ``MODULE_EXT_ROOT`` can be described as:
 This allows control of the build inclusion to be described externally to the
 Zephyr module.
 
+SBOM integration
+====================
+
+The module description file :file:`zephyr/module.yml` can be used to improve the
+generated SBOM and vulnerability monitoring.
+
+The version and the CPE associated to the module can be provided using
```

Comment:
when using abbreviations like: `CPE` and `CVE` (and soon `VEX`), we should write them in full once, so that even users unfamiliar with the terms know what they mean.